src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/ccd/CaseDataHelper.java

```diff
@@ -38,17 +38,29 @@ public Long createCase(Map<String, Object> sscsCaseData, String userAuthToken, S
         return caseDetails.getId();
     }
 
-    private CaseDataContent caseDataContent(Map<String, Object> sscsCaseData, String eventToken, String eventId) {
+    public void updateCase(Map<String, Object> sscsCaseData, String userAuthToken, String serviceAuthToken, String userId, String eventId, Long caseId, String summary, String description) {
```

Comment:
Doesn't this method exist in the commons library which can be reused?

Reply:
No it's slightly different in common library. Bulk scan doesn't have the SscsCaseData object, instead it just has key value pairs.